### ISSUE TITLE
Date to deliver by

### ISSUE THREAD

**Issue by sm2x** [openboxes/openboxes#451]
How do you add this field on Requisitions, so as to accept future date of delivery?

**Comment by jmiranda** [comment]
The field currently exists on Requisition and is called requestedDeliveryDate. I can add it to the initial "Create Requisition" form if you'd like. Is there any logic that you'd like to see included that would be triggered by the value of that field? For example, I could image that you might want to show some indication or alert someone if the requisition has not been delivered by that date.

**Comment by jmiranda** [comment]
Yeah that date is set to today's date when the object is created and then ignored.
